test/kitchen_sink/kitchen_sink.c

```diff
@@ -5,18 +5,44 @@
  */
 
 #include <stdio.h>
-#include "pico/stdlib.h"
-#include "pico/time.h"
+// Include all headers to check for compiler warnings
```

Comment:
Is it worth having a build-time script do this automatically, as it's likely that additional headers will get added to the SDK in future?

Reply:
Partially added in #319 :slightly_smiling_face: